static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -305,9 +316,7 @@ const MemberCardContentRow = styled('div')`
   align-items: center;
   margin-bottom: ${space(0.25)};
   font-size: ${p => p.theme.fontSizeSmall};
-  & > *:first-child {
-    margin-right: ${space(0.75)};
-  }
+  gap: ${space(0.75)};
 `;
```

Comment:
Should this have been `ButtonBar`?

Reply:
i made the ButtonContainer a `ButtonBar`